.env.example

```diff
@@ -1,7 +1,3 @@
-DOMAIN=example.ch
-BUCKET_NAME=bucket
```

Comment:
Are you sure this is gone ? in the readme it's needed to be set

Reply:
These env variables are no longer needed because the BUCKET_NAME must be provided in the api request.